dbt_platform_helper/commands/environment.py

```diff
@@ -342,6 +342,7 @@ def generate(name, vpc_name):
         )
         raise click.Abort
 
+    session = get_aws_session_or_abort()
```

Comment:
At risk of opening up a can of worms, because I've just realised we don't for any of the other places this method is used, should we have a test to cover this?

It could be just an assert that the method has been called and we rely on [the tests for the method](https://github.com/uktrade/platform-tools/blob/main/tests/platform_helper/utils/test_aws.py) to cover the detail.